cmd/geth/accountcmd.go

```diff
@@ -300,12 +325,7 @@ func accountUpdate(ctx *cli.Context) error {
 	if ctx.Args().Len() == 0 {
 		utils.Fatalf("No accounts specified to update")
 	}
-	stack, _ := makeConfigNode(ctx)
-	backends := stack.AccountManager().Backends(keystore.KeyStoreType)
-	if len(backends) == 0 {
-		utils.Fatalf("Keystore is not available")
-	}
-	ks := backends[0].(*keystore.KeyStore)
+	ks := makeKeyStore(ctx)
```

Comment:
I'm not sure about this. The previous code is more high-level, it starts an account manager and asks it "is there a keystore here". And it might be that the account manager has an external signer, or has only USB signers, so there's no actual keystore. 
The new code ignores all that, and just uses the directory where a keystore would/should reside. I am not sure that is an improvement, since we ideally do not want geth to handle keys.